js/main.js

```diff
@@ -0,0 +1,76 @@
+/*
+ *  Grid drawing
+ */
+
+var canvas = document.querySelector(".canvas");
+var ctx = canvas.getContext("2d");
+ctx.canvas.height = window.innerHeight - 4;  // slight offset or else it overflows and causes scroll in Firefox
+ctx.canvas.width = window.innerWidth;
+
+// Grid parameters
+var pixelSize = 25;
+var borderSize = 2;      // Border is drawn on the right and bottom edges of the pixel
+ctx.fillStyle = "gray";  // This will be the border colour
+ctx.fillRect(0, 0, ctx.canvas.width, ctx.canvas.height);
```

Comment:
(Subjective) 
I've generally been taught to put comments on the line above code, rather than on the same line.

Reply:
Comments reorganized, yeah it's more readable when above for these